packages/storybook/src/stories/notify/notify.stories.mdx

```diff
@@ -0,0 +1,192 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Button, Notify } from '@viamrobotics/prime-core';
+import NotifySlot from './notify.spec.svelte';
+
+<Meta
+  title='Elements/Notify'
+  parameters={{
+    actions: {
+      handles: ['close'],
+    },
+  }}
+  argTypes={{
+    variant: {
+      control: { type: 'select' },
+      options: ['danger', 'warning', 'success', 'info'],
+      description: 'Notification type',
+      table: { defaultValue: { summary: 'info' } },
+    },
+    title: {
+      description: 'Title',
+      table: { defaultValue: { summary: '' } },
+    },
+    message: {
+      description: 'Message',
+      table: { defaultValue: { summary: '' } },
+    },
+  }}
```

Comment:
[opt] You could add an argType for your close event similar to how Ethan does it in the pill story he wrote:

Something like:

`'on:close': {
      description: 'Event fired when user presses close button.',
    },`